app/templates/components/paper-dialog.hbs

```diff
@@ -5,7 +5,7 @@
     fixed=(unless parent true)
     class="md-dialog-backdrop"
     onClick=(action "outsideClicked")}}
-  {{#paper-dialog-container outsideClicked=(action "outsideClicked")}}
+  {{#paper-dialog-container class=(readonly dialogContainerClasses) outsideClicked=(action "outsideClicked")}}
```

Comment:
IMO:
  - `class` for `paper-dialog-inner`, since this is the dialog itself
  - `dialogContainerClass` for `paper-dialog-container`

paper-select component renders two things, a trigger and a dropdown. That's why we need to be more specific.
In this case this is just a "proxy" to render a single component, so there shouldn't be any confusion.